pyxem/signals/indexation_results.py

```diff
@@ -147,6 +161,516 @@ def _get_second_best_phase(z):
+                ragged=True,
+                output_dtype=object,
+                output_signal_size=(),
+                navigation_chunks=navigation_chunks,
+            )
+            markers = hs.plot.markers.Points.from_signal(
+                coords, facecolor="none", edgecolor=color, **kwargs
+            )
+            yield markers
+
+            if annotate:
+                texts = vectors.map(
+                    vectors_to_text,
+                    inplace=False,
+                    lazy_output=lazy_output,
+                    ragged=True,
+                    output_dtype=object,
+                    output_signal_size=(),
+                )
+                # New signal for offset coordinates, as using inplace=True shifts the point markers too
+                text_coords = coords.map(lambda x: x + annotation_shift, inplace=False)
+                text_markers = hs.plot.markers.Texts.from_signal(
+                    text_coords, texts=texts.data.T, color=text_color, **text_kwargs
+                )
+                yield text_markers
+
+    def to_single_phase_polar_markers(
+        self, signal_axes: AxesManager, n_best: int = 1, marker_color: str = "red"
+    ) -> Iterator[hs.plot.markers.Markers]:
+        (
+            r_templates,
+            theta_templates,
+            intensities_templates,
+        ) = self.simulation.polar_flatten_simulations(
+            signal_axes[1].axis, signal_axes[0].axis
+        )
+
+        def marker_generator_factory(n_best_entry: int, r_axis, theta_axis):
+            theta_min, theta_max = theta_axis.min(), theta_axis.max()
+
+            def marker_generator(entry):
+                index, _, rotation, mirror = entry[n_best_entry]
+                index = index.astype(int)
+                mirror = mirror.astype(int)
+
+                r_ind = r_templates[index]
+                r = r_axis[r_ind]
+
+                theta_ind = theta_templates[index]
+                theta = theta_axis[::mirror][theta_ind] + np.deg2rad(rotation)
+
+                # Rotate as per https://github.com/pyxem/pyxem/issues/925
+                theta += np.pi
+
+                # handle wrap-around theta
+                theta -= theta_min
+                theta %= theta_max - theta_min
+                theta += theta_min
+
+                mask = r != 0
+                return np.vstack((theta[mask], r[mask])).T
+
+            return marker_generator
+
+        for n in range(n_best):
+            markers_signal = self.map(
+                marker_generator_factory(n, signal_axes[1].axis, signal_axes[0].axis),
+                inplace=False,
+                ragged=True,
+                lazy_output=True,
+            )
+            markers = hs.plot.markers.Points.from_signal(
+                markers_signal, facecolor="none", edgecolor=marker_color, sizes=(10,)
+            )
+            yield markers
+
+    def to_navigator(self, direction: Vector3d = Vector3d.zvector()):
```

Comment:
Maybe we should call this `to_IPF_colormap` instead, as using them as a navigator is just an option. We could also add an option to return the lookup colormap too, as it could technically use a different mapping than what the user might expect. 

Something like 
~~~python
if return_colormap:
   return s, ipfcolorkey.plot(return_figure=True)
return s
~~~

Reply:
Sounds like a good idea.  It might be nice to include a marker related to the IPF color map. Let me work on that!

Reply:
This is now done